core-blocks/image/index.js

```diff
@@ -190,6 +199,9 @@ export const settings = {
 				const { url, alt, caption, align, href, width, height } = attributes;
 				const extraImageProps = width || height ? { width, height } : {};
 				const image = <img src={ url } alt={ alt } { ...extraImageProps } />;
+				const classes = classnames( align ? `align${ align }` : null, {
+					'is-resized': !! width || !! height,
+				} );
```

Comment:
I don't understand this change? I think the deprecated versions shouldn't be updated once they land. I think what we need to do here is to add a new `deprecated` version with the `save` function without the `is-resized` className. That way the blocks using that version would be considered as valid.

Reply:
Sounds good to me. Can you help me with this? Feel free to push directly

Reply:
Sounds good to me. Can you help me with this? Feel free to push directly